.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Can we add prerelease branches here too?

Reply:
Oh, yes. I forgot about the base branch prerequisite. Do we want to limit what PRs this runs on? I think I'll just remove it.